CONTRIBUTORS.md

```diff
@@ -846,7 +846,7 @@
 * Ankit Kumar
 * Frank Yiu
 * Shaurya Panchal
-
+* Srishti Jaiswal
 ## Translators
```

Comment:
Hi @Srishti-j18 - for future reference, there's no need to add your name here. We do it during the merge fo avoid conflicts.

Reply:
Okay, I'll keep that in mind. For now, should I remove my name from here?

Reply:
I have removed this commit.